examples/howto/plot_record_extracellular_potentials.py

```diff
@@ -86,24 +86,17 @@
                         gridspec_kw={'height_ratios': [1, 3, 2]})
 
 # Then plot the aggregate dipole time series on its own axis
-dpl[trial_idx].copy().smooth(
-    window_len=window_len).plot(ax=axs[0], decim=decimate,
-                                show=False)
-
-voltage_offset = 50  # the spacing between individual traces
-voltage_scalebar = 200  # can be different from offset
-# we can assign each electrode a unique color using a linear colormap
-colors = plt.get_cmap('cividis', len(electrode_pos))
+dpl[trial_idx].smooth(window_len=window_len)
+dpl[trial_idx].plot(ax=axs[0], decim=decimate)
 
 # use the same smoothing window on the LFP traces to allow comparison to dipole
 net.rec_arrays['shank1'][trial_idx].smooth(window_len=window_len).plot(
-    ax=axs[1], contact_labels=depths, color=colors, decim=decimate, show=False,
-    voltage_offset=voltage_offset, voltage_scalebar=voltage_scalebar)
+    ax=axs[1], decim=decimate)
```

Comment:
I don't know why `show=False` is set in all our examples ... it seems not necessary as far as I can tell.

Reply:
I've run into situations where not setting it to `False` messes things up in multipanel figures. Haven't investigated too deeply but I would hesitate to change this,

Reply:
let's keep it like this in this one example and I'll let you complain if you run into issues ;-) I've seen this work in MNE for a long time, so I'm surprised it causes issues

Reply:
also -- if `show=False` is indeed better, then that should be default. Ultimately the example should be kept simple and expose arguments only with intention.

Reply:
I think this is what's causing the CI tests to fail. Without setting `show=False`, it renders the figure every time because `plt.show()` is called inside each `plot_...` function.

+1 for setting `show=False` as the default value for all plot functions in `viz.py`.

Reply:
Side note, I've also encountered this issue numerous times when trying to plot stuff as a user.

Reply:
okay, so 

https://jakevdp.github.io/PythonDataScienceHandbook/04.00-introduction-to-matplotlib.html#show()-or-No-show()?-How-to-Display-Your-Plots

says there should be only one `plt.show()` command. Fair enough ... I'll revert :)